scripts/linux.sh

```diff
@@ -15,6 +15,11 @@ get_mountpoints() {
   grep "^$1" /proc/mounts | cut -d ' ' -f 2 | sed 's,\\040, ,g' | sed 's,\\011,\t,g' | sed 's,\\012,\\n,g' | sed 's,\\134,\\\\,g'
 }
 
+get_path_id() {
+  # udevadm test-builtin path_id /sys/block/sda | grep ID_PATH=
+  find -L /dev/disk/by-path/ -samefile "$1" | cut -c 19-
```

Comment:
Hmmm, just noticed that on my Ubuntu 14.04 laptop (Linux kernel 3.19.0-82-generic, udev 204-5ubuntu20.28) I don't have a `/dev/disk/by-path/` directory :-/

```bash
$ ls -l /dev/disk
total 0
drwxr-xr-x 2 root root 360 Apr 27 00:40 by-id
drwxr-xr-x 2 root root 120 Apr 27 00:40 by-label
drwxr-xr-x 2 root root  80 Apr 27 00:40 by-partlabel
drwxr-xr-x 2 root root 180 Apr 27 00:40 by-partuuid
drwxr-xr-x 2 root root 180 Apr 27 00:40 by-uuid
```